code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -227,23 +225,61 @@ actually very simple. To translate a plain English sentence to Ratvar's tongue,
+	There are, however, a few parts of the Ratvarian tongue that aren't typical and are implemented for fluff reasons. Some words may have graves, or hyphens (prefix and postfix), making the plain
 English translation apparent but disjoined (for instance, "Orubyq zl-cbjre!" translates directly to "Behold my-power!") although this can be ignored without impacting overall quality. When
 translating from Ratvar's tongue to plain English, simply remove the disjointments and use the finished sentence. This would make "Orubyq zl-cbjre!" into "Behold my power!" after removing the
 abnormal spacing, hyphens, and grave accents.
 
 List of nuances:
-
 - Any time the word "of" occurs, it is linked to the previous word by a hyphen. (i.e. "V nz-bs Ratvar." directly translates to "I am-of Ratvar.")
-- Any time "th", followed by any two letters occurs, you add an apostrophe between those two letters, i.e; "Thi's"
-- In the same vein, any time "ti", followed by one letter occurs, you add an apostrophe between "i" and the letter, i.e; "Ti'me"
+- Any time "th", followed by any two letters occurs, you add a grave (`) between those two letters, i.e; "Thi`s"
+- In the same vein, any time "ti", followed by one letter occurs, you add a grave (`) between "i" and the letter, i.e; "Ti`me"
 - Whereever "te" or "et" appear and there is another letter next to the e(i.e; "m"etal, greate"r"), add a hyphen between "e" and the letter, i.e; "M-etal", "Greate-r"
-- Where the word "and" appears, it is linked to the previous and following words by hyphens, i.e; "Sword-and-shield"
+- Where the word "and" appears surrounded on both sides by other words, it is linked to those words by hyphens, i.e; "Sword-and-shield", but not "And this?"
 - Where the word "to" appears, it is linked to the following word by a hyphen, i.e; "to-use"
 - Where the word "my" appears, it is linked to the following word by a hyphen, i.e; "my-light"
-
 - Although "Ratvar" translates to "Engine" in English, the word "Ratvar" is used regardless of language as it is a proper noun.
  - The same rule applies to Ratvar's four generals: Nezbere (Armorer), Sevtug (Fright), Nzcrentr (Amperage), and Inath-Neq (Vangu-Ard), although these words can be used in proper context if one is
    not referring to the four generals and simply using the words themselves.
-
 */
+
+//Regexes used to alter english to ratvarian style
+#define RATVAR_OF_MATCH			regex("(?<=\\w)\\s(\[oO]\[fF])","g")
+#define RATVAR_OF_REPLACEMENT 	"-$1"
+#define RATVAR_TH_MATCH			regex("(\[tT]\[hH]\\w)(\\w)","g")
+#define RATVAR_TH_REPLACEMENT	"$1`$2"
+#define RATVAR_TI_MATCH			regex("(\[tT]\[iI])(\\w{2})","g")
+#define RATVAR_TI_REPLACEMENT	"$1`$2"
+#define RATVAR_ET_MATCH			regex("(\[eE]\[tT])(\\w)","g")
+#define RATVAR_ET_REPLACEMENT	"-$1$2"
+#define RATVAR_TE_MATCH			regex("(\\w)(\[tT]\[eE])","g")
```

Comment:
I don't actually know regex, but this looks like it's checking for a letter next to the T and not next to the E?